packages/eslint-plugin/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
```

Comment:
This should probably include `"noEmit": true`, otherwise running `tsc` from `packages/eslint-plugin` returns the following errors.

![image](https://user-images.githubusercontent.com/357832/120373135-d7870880-c2cc-11eb-929f-e18db00a9a34.png)